yml/dp02_dc2.yaml

```diff
@@ -1,6 +1,6 @@
 ---
-name: dp02Schema
-"@id": "#dp02Schema"
+name: dp02_dc2_catalogs
+"@id": dc2
```

Comment:
Following this line, please add

`description: Preliminary schema for Data Preview 0.2, including only catalog tables for now`